Add UIAPicker and UIAPickerWheel to UIAModels (needed for type criteria)

johnhammerlund opened the pull request.

johnhammerlund committed:
Add UIAPicker and UIAPickerWheel to UIAModels (needed for type criteria)

lukeis commented:
Mind updating the year in the copyright? :) 

Also I'm not sure what these are for, I'll have to look at the rest of the code

johnhammerlund committed:
Add UIAPicker and UIAPickerWheel to UIAModels (needed for type criteria)

johnhammerlund commented:
Had a bit of an internet issue so the commit message wasnt properly ammended, but I updated the copyright :p and I've implemented a wrapper for Pickers for my baseline tests, but the problem was associated with https://github.com/ios-driver/ios-driver/blob/dev/server/src/main/java/org/uiautomation/ios/server/command/uiautomation/BaseFindElementNHandler.java#L117-124

Basically, `driver.FindElement(By.ClassName("UIAPicker"));` would throw a ClassNotFoundException.
